### PR TITLE
Netbox v3 support

### PR DESCRIPTION
**Pull request by YWatchman**
This PR covers:

- [x] Installable in Netbox v3;
- [x] Templates with new Netbox frontend
- [x] Tested v3
- [ ] Tested v2.11.12
- [ ] Tested v2.10.6

### CODE REVIEW

**Comment by jozefrebjak** [comment]
I hope that this plugin will work in the newest version of netbox 👌

**Comment by YWatchman** [comment]
@jozefrebjak it does! Wait until this gets merged, shortly.

File: netbox_ddns/tables.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
     {% if perms.netbox_ddns.change_extradnsname %}
         <a href="{% url 'plugins:netbox_ddns:extradnsname_edit' ipaddress_pk=record.ip_address.pk pk=record.pk %}" 
            class="btn btn-xs btn-warning">
-            <i class="glyphicon glyphicon-pencil" aria-hidden="true"></i>
+            <i class="mdi mdi-pencil" aria-hidden="true"></i>
```
Comment (YWatchman) [inline netbox_ddns/tables.py:19]: This is incompatible with Netbox < 3.0.0

File: netbox_ddns/templates/netbox_ddns/ipaddress/dns_refresh_button.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {% if perms.ipam.change_ipaddress %}
-    <form method="post" style="display: inline-block">
+    <form method="post" class="inline-block">
         {% csrf_token %}
 
         <button type="submit" name="_edit"
                 formaction="{% url 'plugins:netbox_ddns:ipaddress_dnsname_recreate' ipaddress_pk=object.pk %}"
-                class="btn btn-default">
-            <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span> Recreate DNS
+                class="btn btn-sm btn-secondary">
```
Comment (YWatchman) [inline netbox_ddns/templates/netbox_ddns/ipaddress/dns_refresh_button.html:7]: class `btn-xs` does not exist in this version of bootstrap.

**Pull request merged**